contracts/upgradeable/mocks/extensions/soulbound/RMRKSoulboundEquippableMockUpgradeable.sol

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: Apache-2.0
+
+pragma solidity ^0.8.18;
+
+import "../../../../RMRK/extension/soulbound/IERC6454.sol";
+import "../../RMRKEquippableMockUpgradeable.sol";
+
+contract RMRKSoulboundEquippableMockUpgradeable is
```

Comment:
I'm guessing there was no way to reuse the abstract implementations of Soulbound. Is that correct?
If so, is it worth having them at all?
Maybe core shouldn't have to receive name and symbol? This is something which has bugged me for a while..  It could be split into core and metadata extension, and might solve this problem (not 100% sure).  Not that I would do the change on this PR anyway.